sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsClient.java

```diff
@@ -457,4 +457,12 @@ private Map<String, ParticipantProperties> convertRoomIdentifiersToMapForRemove(
 
         return participantMap;
     }
+
+    private static String getUpdateRequest(Map<String, ParticipantProperties> participantMap) throws IOException {
+        Writer json = new StringWriter();
+        try (JsonWriter jsonWriter = JsonProviders.createWriter(json)) {
+            new UpdateParticipantsRequest().setParticipants(participantMap).toJson(jsonWriter);
```

Comment:
same here, please call flush (and consider the suggestion)

Reply:
I thought the JsonWriter already class flush on close.
Here is the ref: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/serialization/azure-json/src/samples/java/com/azure/json/codesnippets/WritingJsonExamples.java#L66

Reply:
ah, cool, thanks!